File: Java ejs/UD1/UD01A018.java
public class UD01A018{
	public static void main(String[] args){
		
		float IVA=21f;
		int precioBruto=1000;
		float precioNeto;
		precioNeto=precioBruto/(1+IVA/100);
		System.out.printf("Precio neto (sin IVA): "+ precioNeto);
		System.out.printf("IVA (21%): "+(float) (precioBruto-precioNeto));
		System.out.printf("Precio total (con IVA): "+precioBruto);
	}
}

File: Python ejs/UD01/A01/UD01A018.py
iva=21
precioBruto=1000
precioNeto=float(precioBruto/(1+iva/100))
print("Precio neto (sin IVA): " + str(precioNeto))
print("IVA (21%): "+ str(float(precioBruto-precioNeto)))
print("Precio total (con IVA): "+ str(precioBruto))